api/src/controllers/reviews.ts

```diff
@@ -220,6 +220,44 @@ router.patch("/getVoteColor", async function (req, res) {
   }
 });
 
+/**
+ * Get multiple review colors
+ */
+router.patch("/getVoteColors", async function (req, res) {
+  if (req.session.passport != null) {
+    //query of the user's email and the review id
+    let ids = req.body["ids"];
+    let colors = [];
+    for (let i = 0; i < ids.length; i++) {
```

Comment:
You can use the [$in](https://www.mongodb.com/docs/manual/reference/operator/query/in/) operator to do this in one query.